pkgs/development/python-modules/python-unshare/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "python-unshare";
+  # pypi version doesn't support Python 3 and the package didn't update for a long time:
+  # https://github.com/TheTincho/python-unshare/pull/8
+  version = "2018-05-20";
```

If i remember correctly, the preferred practices is to prefix with unstable when it's not an official release
```suggestion
  version = "unstable-2018-05-20";
```

---

Didn't know that, I see it in https://nixos.org/nixpkgs/manual/#sec-package-naming now. Thanks!